consensus/mod.go

```diff
@@ -53,6 +53,8 @@ type Actor interface {
 	// Propose performs the consensus algorithm using the list of nodes
 	// as participants.
 	Propose(proposal Proposal, players mino.Players) error
+
+	Close() error
```

Comment:
Should the close be used to signal a normal ending operation or an unexpected event? In the later case "Abort()" would be more appropriate.

Reply:
No it's for normal close.